packages/yoastseo/src/scoring/assessments/index.js

```diff
@@ -35,7 +34,6 @@ import ProductSKUAssessment from "./seo/ProductSKUAssessment";
 import InclusiveLanguageAssessment from "./inclusiveLanguage/InclusiveLanguageAssessment";
 
 const readability = {
-	ListAssessment,
```

Comment:
List Assessment is currently only used in product assessors. Hence, it's not necessary to bundle this assessment with the others that will be exposed in Free.